docs/contributing.rst

```diff
@@ -257,5 +257,12 @@ without warnings. You can do that locally with::
 Review process
 --------------
 
-Pull requests will be reviewed by a project maintainer. All changes to *sgkit* require
-approval by at least one maintainer.
+All pull requests require a pre-existing issue. If you plan to submit a PR,
+please create an issue first to describe and discuss your idea/plan/problem.
```

Comment:
Maybe add the bit about mentioning the issue in the PR comments here from https://github.com/pystatgen/sgkit/issues/125

Reply:
Done